doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -26,12 +26,22 @@ An OSD's status is either in the cluster (``in``) or out of the cluster
 running (``down``). If an OSD is ``up``, it may be either ``in`` the cluster
 (you can read and write data) or it is ``out`` of the cluster. If it was
 ``in`` the cluster and recently moved ``out`` of the cluster, Ceph will migrate
+
+THIS PART IS INCONSISTENT OR AT THE VERY LEAST, CONFUSING.
+
+ORIGINAL TEXT CONTINUES HERE
 placement groups to other OSDs. If an OSD is ``out`` of the cluster, CRUSH will
 not assign placement groups to the OSD. If an OSD is ``down``, it should also be
```

Comment:
@rzarzynski: Can you take a look at this part of the text, and help us clear it up?

Reply:
`out` has always troubled me a bit, if it's not in cluster, where is it?  A long way down the road to the chemist's?

`up and running` sounds like an O'Reilly book, but a bit redundant and tail-recursive here.

I struggle to find an ideally cogent phrasing, but  might suggest something akin to

>An OSD's status is either_ in_ service (``in``) or _out_ of service
(``out``); and, it is either running and reachable (``up``), or it is not running or not reachable (``down``). If an OSD is ``up``, it may be either ``in`` service 
(clients can read and write data) or it is ``out`` of service. If an OSD is ``in`` but then due to a failure or manual action is set to the ``out`` state, Ceph will migrate
placement groups to other OSDs to maintain the configured redundancy.  Stick THAT in Kafka's pipe and smoke it ;)  If an OSD is ``out`` of service, CRUSH will
not assign placement groups to it. If an OSD is ``down``, it will also be ``out`` (the classic "down and out") unless you have done something "cute".


I really REALLY want to say "in order" or "out of order" to set up a pun about how `ceph status` should report "massive ∆S" ;)  Go look up the units in which entropy is measured and once your brain stops bleeding you'll chuckle.  Or change my contributor attribution to `esel`.